tools/snippet-testing/snippet_tester.rb

```diff
@@ -218,6 +228,10 @@ def parse_options(args)
       options.source_folder = File.expand_path(dir)
       options.test_default = true if options.test_default.nil?
     end
+
+    opts.on('-h', 'Append the twilio endpoints to your hosts file for testing') do
```

Comment:
Good catch, I was doubting about it too.
I did a few changes and now I'm using `-u` flag, does that sound good to you?